packages/platform-sdk-profiles/src/plugins/plugin-repository.ts

```diff
@@ -44,12 +45,24 @@ export class PluginRepository {
 		this.#data.set(`${plugin.id}`, plugin);
 	}
 
-	public fill({ data, blacklist }: { data: object; blacklist: string[] }): void {
+	public fill({
+		data,
+		blacklist = [],
+		whitelist = [],
+	}: {
+		data: object;
+		blacklist?: string[];
+		whitelist?: string[];
+	}): void {
```

Comment:
@faustbrian making them optional (both blacklist and whitelist) that default to empty sets, to prevent profile restoration from throwing "not iterable" error when encrypted/encoded data do not have these set already.